tests/metadata/v1/alert.json

```diff
@@ -2,6 +2,7 @@
   "properties": [ "id", "message", "userId", "timestamp"],
   "actions": [
     "get",
+    "create",
```

Comment:
why not using a existing model that already have create? If there is none then the create test should be added to the alert tests.